tests/queries/0_stateless/02970_system_reload_asynchronous_metrics.sql

```diff
@@ -0,0 +1 @@
+SYSTEM RELOAD ASYNCHRONOUS METRICS;
```

Comment:
you can make a much better test
set `asynchronous_metrics_update_period_s` to a really large value
and then pick a metric, e.g. `Uptime` or `NumberOfTables` to check in `system.asynchronous_metrics` it's updated after this call

Reply:
Done, thanks.